src/InvokePlaster.ps1

```diff
@@ -830,7 +830,7 @@ function Invoke-Plaster {
                 if (![string]::IsNullOrWhiteSpace($description)) {
                     $newModuleManifestParams['Description'] = $description
                 }
-                if (![string]::IsNullOrWhiteSpace($psVersion)) {
+                if (![string]::IsNullOrWhiteSpace($PowerShellVersion)) {
```

Comment:
For consistency, can you change the casing to camelCase (powerShellVersion) here and on line 770 where it is assigned?